life-and-letters/original/08-chapter-8.adoc

```diff
@@ -683,6 +683,9 @@ About this time she penned the following letter, viz.
 [.signed-section-context-open]
 To a Friend of Ireland, written a short time before I left it, in the year 1776.
 
+[.embedded-content-document.letter]
```

Comment:
this one also (maybe it's the same one, just in the original?) got inserted to low, it should be above the context-open chunk above.